### DISCUSSION TITLE
Install adonis with one line

### DISCUSSION THREAD

**Discussion by JonhnyDev** [adonisjs/core#3778] [General]
Hi guys, I'm using ansible and terraoform to automatically create my "IAC" servers, all I need is adonisJS5, but during the installation adonis asks some questions like "project name: some", "type: api, web or slim" and so on, in the old days you could pass as settings on the same line --name, --type (--api-only) and etc.

![image](https://user-images.githubusercontent.com/43152652/176051001-d2cbe792-2a13-4520-8b24-8360bb720708.png)

I don't want the screen to appear, I want to pass all variables in the same create command, is it possible?

how to make the complete installation of Adonis in a single line in the same shell was possible in AdonisJS4 ?

**Comment by thetutlage** [comment]
Can you still pass those options as flags. Check the README of the installer repo https://github.com/adonisjs-community/create-adonis-ts-app#options

**Comment by JonhnyDev** [comment]
perfect.
